README.md

```diff
@@ -255,8 +255,8 @@ These instructions assume Jenkins, yarn, TypeScript, mocha and eslint.
       }
     
       stage("publish") {
-        sh "yarn beehive-flow publish"
-        sshagent(credentials: ['jenkins2-github']) {
+        sshagent(credentials: ['my-git-credentials']) {
+          sh "yarn beehive-flow publish"        
```

Comment:
Do we need to update the README to recommend this change as well?

Reply:
This is the readme. I forgot to update beehive-flow's Jenkinsfile - I've done that now.

Reply:
:facepalm: sorry I knew it was only done in one location, but seems I got them mixed up.